Restrict share by partition (#67)
canonical/awspub: docs/how_to/publish.rst

  awspub create config.yaml
  awspub publish config.yaml

Sharing images
~~~~~~~~~~~~~~

> Images can be shared with other AWS accounts. For that, the account ID is required.

Reviewer: ```suggestion
Images can be shared with other AWS accounts. For that, the account IDs of the other accounts are needed.
```

.. literalinclude:: ../config-samples/config-minimal-share.yaml
   :language: yaml

> In that example, the image `my-custom-image` will be share with the account `1234567890123`

Reviewer: ```suggestion
In the above example, the image `my-custom-image` will be shared with the account `123456789123`
```

> when `awspub` runs in the commercial partition (`aws`, the default) and will only be shared

Reviewer: ```suggestion
when `awspub` runs in the commercial partition (`aws`, the default). It'll be shared
```

> with the account `456789012345` when `awspub` runs in the the china partition (`aws-cn`).

Reviewer: ```suggestion
with the account `456789012345` when `awspub` runs in the the china partition (``aws-cn``).
```
By putting aws-cn in double back-ticks you can exclude it from the spelling check.
